### ISSUE TITLE
Does not work on IE 10 (maybe other versions too)

### ISSUE THREAD

**Issue by yyunikov** [mudassir0909/stackoverflow-card#7]
When I test this on IE 10, I get the following error in script:
<code>
SCRIPT5007: Unable to get property 'userid' of undefined or null reference 
so-card-widget.min.js, line 4 character 1647

Here is how the div look like:

``` html
<div id="so-card-widget" data-userid="1889928" data-theme="default"></div>
```

The script I'm using:

``` html
<script type="text/javascript" src="//mudassir0909.github.io/stackoverflow-card/dist/1.0.0/so-card-widget.min.js"></script>
```

Any suggestions?

**Comment by mudassir0909** [comment]
Looks like [Element.dataset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement.dataset) is not supported in IE < 11. [This answer](http://stackoverflow.com/questions/7127896/problem-using-elem-dataset-with-ie-and-jsfiddle) should fix this problem. 

Can you create a PR for this ? If not then I'll fix this during weekend

**Comment by yyunikov** [comment]
Yes, I have created [PR](https://github.com/mudassir0909/stackoverflow-card/pull/8). Please take a look.

**Comment by mudassir0909** [comment]
Cool, Thanks! merging it right away

**Comment by mudassir0909** [comment]
reopening, one minor change needed

**Comment by yyunikov** [comment]
Ok, no problem, I think min version should be generated with Grunt as well. Probably I've commited in wrong directory.

**Comment by mudassir0909** [comment]
yes min version will be generated by grunt
